test/kubernetes/deploy/README.md

```diff
@@ -0,0 +1,53 @@
+# Deploying and Testing a Development Version
+
+The most convenient way to test changes to csi-digitalocean is on a DOKS
```

Comment:
Link to the product page for the first instance of "[DOKS](https://www.digitalocean.com/products/kubernetes/)"?

Reply:
Good idea - expanded DOKS to DigitalOcean Kubernetes in the first instance, and also added the link.